### ISSUE TITLE
Error setting up the repo - AuthToken is not being generated.

### ISSUE THREAD

**Issue by ujwalkumar1995** [calcom/cal.com#11365]
### Issue Summary

I am facing issues while running the repo in local. I am able to setup the code base but when trying to login I am getting an error in the login flow. I have attached screenshot of the logs for reference. On further debugging, it seemed that the token is not being generated from getServerSession. My local setup seems to be fine. I have been working on the repo for quite sometime now. This started happening yesterday as I had to wipe my system data and do a fresh setup of the repo. Since then it has started popping up. I am not sure if this issue is restricted to my local system or is it a wider issue, therefore raising it here to seek help.

### Steps to Reproduce

- Do a fresh clone of the repo.
- Setup the repo following the Readme.
- Try to login from any user seeded as part of the local setup.

### Actual Results

Getting unauthorized error.

### Expected Results

Should have allowed me to login.

### Technical details

- Browser version, screen recording, console logs, network requests:  Chrome (Version 116.0.5845.188)
- Node.js version - 16.16.0

### Evidence

Attached screenshot of error log.

![Screenshot 2023-09-14 210248](https://github.com/calcom/cal.com/assets/20976813/867f2695-ca56-4d31-a5cf-60742c008eef)

**Comment by PeerRich** [comment]
have you tried doing `yarn dx` with docker running?

**Comment by ujwalkumar1995** [comment]
Hey @PeerRich 
Yes, I had docker running when I did yarn dx. I am able to see the login page as well. It just won't allow me login from any user. I have confirmed the users have been populated in the database. 

I have been contributing to the repo since couple of months and have setup the repo from scratch multiple times. I had to do a fresh setup couple of days back and since then this issue has popped up. Not sure if some dependency got updated recently that could have caused this.
